src/main/resources/application.properties

```
@@ -22,4 +22,5 @@ cloud.aws.credentials.accessKey=${AWS_ACCESS_KEY_ID}
 cloud.aws.credentials.secretKey=${AWS_SECRET_ACCESS_KEY}
 cloud.aws.s3.bucket=hoetfbucket1
 cloud.aws.region.static=ap-northeast-2
-cloud.aws.stack.auto-=false
\ No newline at end of file
+cloud.aws.stack.auto-=false
```

혹시 auto 오른쪽에 짝대기(`-`) 빠지는 게 맞는 거라면 수정 부탁드려요.
```suggestion
cloud.aws.stack.auto=false
```